fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -100,8 +101,14 @@ public NereidsPlanner getPlanner() {
         return planner;
     }
 
-    @Override
-    public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+    public Transaction getTxn() {
+        return txn;
+    }
+
+    /**
+     * init plan
```

Comment:
init plan is too simple, the comment is same to the method name, nobody could know the reason.
please add more comment to explain why we need modify this. Then other people could modify it or refactor the code.